### DISCUSSION TITLE
What does Score Limit mean?

### DISCUSSION THREAD

**Discussion by gitbittidbit** [MalloyDelacroix/DownloaderForReddit#337] [Q&A] [answered]
Hi, I don't understand what Score Limit and Score Limit Eval do. Can someone plz explain? Thx

**Comment by MalloyDelacroix** [comment, marked as answer]
The score limit and score limit eval settings let you filter the posts you download by karma score.  If the score limit eval option is set to NO LIMIT, a posts karma score will not be considered in the filtering process.  If score limit eval is set to GREATER THAN, and the score limit is set to 4,000, only posts whose karma score is greater than 4,000 will be downloaded.  If set to LESS THAN, only posts with a karma score of less than 4,000 will be downloaded.
